chains/nobletestnet/metadata.yaml

```diff
@@ -42,3 +42,5 @@ rpcUrls:
   - http: https://rpc.testnet.noble.xyz:443
 slip44: 118
 technicalStack: other
+transactionOverrides:
+  gasPrice: 0.1
```

Comment:
This is the gas price of USDN, which can be confirmed via [this](https://api.testnet.noble.xyz/noble/globalfee/v1/gas_prices) query!